.expeditor/config.yml

```diff
@@ -42,5 +42,5 @@ merge_actions:
         - "Expeditor: Skip All"
   # The markdown generator depends on the updated CHANGELOG.md so
   # should run after it
-  - bash:.expeditor/generate_markdown.sh:
-      only_if: built_in:update_changelog
+  # - bash:.expeditor/generate_markdown.sh:
+  #     only_if: built_in:update_changelog
```

Comment:
Shouldn't we still update the change log for app? And just not produce release notes from it?

Reply:
`CHANGELOG.md` will still be getting updated by `built_in:update_changelog:` on line 38. By commenting out the lines I did we will not convert the local `CHANGELOG.md` into `assets/html/package_details.html`